modules/gitops-deploying-a-rollout.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/getting-started-with-argo-rollouts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-deploying-a-rollout_{context}"]
+= Deploying a rollout
+
+As a cluster administrator, you can configure Argo Rollouts to progressively route a subset of user traffic to a new application version. This allows you to test if the application is deployed and working.
+
+The following example procedure creates a `rollouts-demo` rollout and service. It then routes 20% of traffic to a canary version of the application, waits for a manual promotion, and then performs multiple automated promotions until it routes the entire traffic to the new application version.
```

Comment:
```suggestion
The following example procedure creates a `rollouts-demo` rollout and service. It then routes 20% of traffic to a canary version of the application, waits for a manual promotion, and then performs multiple automated promotions until it routes the entire traffic to the new application version.
```
I don't know what "it" is here. Do we mean the rollout service?

Reply:
No, the rollout. I have addressed it. thanks!